### ISSUE TITLE
[Model] HRED

### ISSUE THREAD

**Issue by hzhwcmhf** [thu-coai/contk#10]
Refer to Building end-to-end dialogue systems using generative hierarchical neural network models

**Comment by hzhwcmhf** [comment]
Dataloader
* Ubuntu

Metric
* Perplexity BLUE

**Comment by hzhwcmhf** [comment]
TODO:
fixing new API

**Comment by hzhwcmhf** [comment]
Done
